metrics/microprofile/src/main/java/com/datastax/oss/driver/internal/metrics/microprofile/MicroProfileMetricUpdater.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.internal.metrics.microprofile;
+
+import com.datastax.oss.driver.api.core.config.DriverExecutionProfile;
+import com.datastax.oss.driver.internal.core.metrics.MetricUpdater;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import net.jcip.annotations.ThreadSafe;
+import org.eclipse.microprofile.metrics.MetricRegistry;
+
+@ThreadSafe
+public abstract class MicroProfileMetricUpdater<MetricT> implements MetricUpdater<MetricT> {
+
+  public static final String CASSANDRA_METRICS_PREFIX = "cassandra";
```

Comment:
This was just the pattern I followed from what we did with the quarkus extension. I can remove it. Just to clarify, removing this would result in metrics paths going from something like:
```
cassandra.s0.cql-requests
```
to something like:
```
s0.cql-requests
```

Reply:
I see. Is it possible to let the user "nest" the driver metrics inside a general prefix of their choice?

Reply:
It is to the extent that a user application can override the Session name. But this implementation is just a fixed static prefix of "cassandra".